modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/util/SortUtil.java

```diff
@@ -60,9 +61,20 @@ public static Sort[] getSorts(
 		for (int i = 0; i < sortFields.size(); i++) {
 			SortField sortField = sortFields.get(i);
 
+			String fieldName = sortField.getSortableFieldName(
```

Comment:
Can we avoid subclassing and casting like this?
```for (int i = 0; i < sortFields.size(); i++) {
			SortField sortField = sortFields.get(i);

			String fieldName = sortField.getSortableFieldName(
				acceptLanguage.getPreferredLocale());

			String fieldPath = sortField.getSortableFieldPath(
				acceptLanguage.getPreferredLocale());

			sorts[i] = new Sort(
				fieldName, fieldPath, Sort.STRING_TYPE,
				!sortField.isAscending());
		}```

Reply:
Done